src/Sulu/Bundle/ContentBundle/Resources/public/js/components/content/settings/main.js

```diff
@@ -377,6 +380,26 @@ define([
                 this.sandbox.dom.attr('#shadow_on_checkbox', 'checked', true);
                 this.sandbox.emit('husky.toolbar.header.item.disable', 'state', false);
             }
+
+            var def = $.Deferred();
+
+            // set author form data
+            this.sandbox.form.create('#settings-author-container').initialized.then(function() {
```

Comment:
the problem is that it would be a quite big refactoring here in this form. in a perfect world everything in the form should be handled in a husky-form.